meadControllerNodeWithWebApp/meadControllerNodeWithWebApp.ino

```diff
@@ -1,3 +1,4 @@
+
 #include "Adafruit_MCP9808.h"
```

Comment:
Fair amount of duplication between this and `meadControllerNodeNoWifi.ino`, would suggest factoring it out